src/types/chart/combo_chart.ts

```diff
@@ -1,11 +1,15 @@
 import { ChartConfiguration } from "chart.js";
 import { Color } from "../misc";
+import { CustomizedDataSet } from "./chart";
 import { ComboBarChartDefinition } from "./common_bar_combo";
 
 export interface ComboChartDefinition extends ComboBarChartDefinition {
+  readonly dataSets: ComboChartDataSet[];
   readonly type: "combo";
 }
 
+export type ComboChartDataSet = CustomizedDataSet & { type?: "bar" | "line" };
```

Comment:
shouldn't it be required ?
```suggestion
export type ComboChartDataSet = CustomizedDataSet & { type: "bar" | "line" };
```

Reply:
NO!